pkg/keystore/keystore.go

```diff
@@ -52,13 +53,20 @@ func New(config *Config) *Keystore {
 	return &Keystore{config}
 }
 
-var defaultConfig = &Config{
-	RootPath:         "/etc/rkt/trustedkeys/root.d",
-	PrefixPath:       "/etc/rkt/trustedkeys/prefix.d",
-	SystemRootPath:   "/usr/lib/rkt/trustedkeys/root.d",
-	SystemPrefixPath: "/usr/lib/rkt/trustedkeys/prefix.d",
+func NewConfig(systemPath, customPath string) *Config {
+	tk := "trustedkeys"
+	root := filepath.Join(tk, "root.d")
+	prefix := filepath.Join(tk, "prefix.d")
+	return &Config{
+		RootPath:         filepath.Join(customPath, root),
+		PrefixPath:       filepath.Join(customPath, prefix),
+		SystemRootPath:   filepath.Join(systemPath, root),
+		SystemPrefixPath: filepath.Join(systemPath, prefix),
+	}
 }
 
+var defaultConfig = NewConfig(common.DefaultSystemConfigDir, common.DefaultCustomConfigDir)
```

Comment:
Uh, no. defaultConfig uses the hardcoded "/etc/rkt/" and "/usr/lib/rkt", while `getKeystore` in rkt/rkt.go creates a keystore config with parameters from cmdline. If no cmdline parameters are specified we indeed end up using the same config as defaultConfig.